interface/globals.php

```diff
@@ -172,6 +174,9 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Includes functions for date internationalization
 include_once (dirname(__FILE__) . "/../library/date_functions.php");
 
+// Includes compoaser autoload
+include_once (dirname(__FILE__) . "/../vendor/autoload.php");
```

Comment:
As an aside, a good fix here would be to convert the two above include_once to require_once .